code/game/machinery/bees_apiary.dm

```diff
@@ -382,7 +382,7 @@
 		//We're low on nutrients, let's call back some bees to reduce our food costs
 		else if (nutrilevel <= 0 && bees_outside_hive.len > 1)
 			for (var/i = 1 to max(1,round(bees_outside_hive.len/3)))
-				var/datum/bee/B = pick(bees_outside_hive)
+				var/datum/bee/B = locate() in bees_outside_hive
```

Comment:
I was otherwise occupied when all this went down. Did the runtime actually break anything, or just clog up the logs? Because if it didn't break anything, it would make more sense to just leave it until the underlying issue is fixed
Alternatively, make it log and continue if it comes across something that shouldn't be there